tests/Unit/Controllers/DriftControllerTest.php

```
@@ -524,6 +524,25 @@ public function testRunDetectionNoDriftSetsFlashMessage(): void
+            'dependency_blocked_max_days' => 7,
+        ])];
+        // fetchAll[0] = baselines (empty), fetchAll[1] = blocked stories (empty)
+        $db = $this->freshDb([$project, $org], [[], []]);
+
+        $request = $this->makePostRequest(['project_id' => '2']);
+        $ctrl = new DriftController($request, $this->response, $this->auth, $db, $this->config);
+        $ctrl->runDetection();
+
+        $this->assertStringContainsString('No drift detected', $_SESSION['flash_message'] ?? '');
+        $this->assertStringContainsString('project_id=2', $this->response->redirectedTo);
+    }
```

PHPMD (2.15.0)

[error] testRunDetectionReadsDependencyBlockedMaxDaysFromOrgSettings accesses the super-global variable $_SESSION. (undefined)

(Superglobals)